content/blog/2019-01-10-vitae.md

```diff
@@ -0,0 +1,122 @@
+---
+slug: "vitae"
+title: "vitae: dynamic CVs with R Markdown"
+authors:
+  - name: Mitchell O'Hara-Wild
+    url: https://mitchelloharawild.com/
+    twitter: mitchoharawild
+date: 2019-01-10
+categories: blog
+tags:
+  - r
+  - vitae
+  - CV
+  - resume
+  - rmarkdown
+  - knitr
+  - unconf
+  - ozunconf
+---
+
+### Why vitae?
+<img src="https://i.imgur.com/PeFoXDy.png" style="width:120px;float:right" />
+The process of maintaining a CV can be tedious. It's a task I often forget about - that is until someone requests it and I find that my latest is woefully out of date. To make matters worse, these professional updates often need repeating across variety of sites (such as ORCID and LinkedIn). 
+
+Having seen several CVs put together into an R Markdown document (including my own, featuring a few quick and dirty hacks to make it work), the need for an R package was obvious. With many attendees of the [2018 rOpenSci OzUnconf](https://ozunconf18.ropensci.org/) having converted their CV to use R Markdown, the conference was the perfect space to develop and formalise the ideas that many of us have worked on independently in the past. Progress on the vitae package progressed much faster than expected, infact it took less time to produce a functional package than it did to tidy the data for the [ozbabynames](https://github.com/ropenscilabs/ozbabynames/) package! The package was primarily developed by myself ([@mitchoharawild](https://twitter.com/mitchoharawild)) and Rob Hyndman ([@robjhyndman](https://twitter.com/robjhyndman)), with ideas/discussion from [many other participants](https://github.com/ropensci/ozunconf18/issues/24).
```

Comment:
The word "package" is used 4 times in 2 sentences starting in "Progress on the vitae package...". Please try to omit a couple. Check the rest of the post for this, and sometimes you could refer to it as `vitae` instead of the `vitae` package each time.